composer.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "shopgate/cart-integration-magento2-shopgate-bolt",
+    "description": "Shopgate bolt module for Magento 2, Extends check_cart functionality to make it possible to prepare preserve and return information about a quote",
+    "type": "magento2-module",
+    "license": "Apache-2.0",
+    "require": {}
```

Comment:
You need to require the shopgate/cart-integration-magento2-export module here, because you are extending files from that module.

Reply:
Just use https://github.com/shopgate/cart-integration-magento2-export/blob/master/composer.json as a reference (also the autoloading part is required)

Reply:
I will add the requirements when the updated versions of the shopgate/cart-integration-magento2-export and shopgate/cart-integration-magento2-base that register the getInternalCartInfo method of the cart helper class are available.

Reply:
The requirements have been added